### PR TITLE
Refactor Managing File Type Content

### PR DESCRIPTION
**Pull request by Lennonka**
Clarification of terminology and workflow.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [x] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5 -- due to conflict, see #1763

### CODE REVIEW

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
I've reviewed your PR against 3.3 before this PR. I assume those comments are also valid here.

**Comment by Lennonka** [comment]
Updated with feedback from #1763

File: guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+endif::[]
+
+.Procedure
+ifdef::satellite[]
+. On your server, ensure that the right repositories are enabled.
+** On {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc:30]:
```suggestion
# subscription-manager repos \
--enable={RepoRHEL8BaseOS} \
```

IMHO nicer to read. See also below.

File: guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+----
+# pulp-manifest /var/www/html/pub/__my_file_repo__
+----
+. Verify the manifest was created:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls /var/www/html/pub/__my_file_repo__
+PULP_MANIFEST test.txt
+----
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc:79]: I am wondering if line 55-79 could be a snippet that's included here and in the other procedure?

**Review by maximiliankolb — APPROVED** [review]
LGTM; made two small comments.

**Comment by Lennonka** [comment]
Cherry-picked:
- d896aba47..a9bef7898  3.5 -> 3.5
- e7e2936ac..5424619aa  3.4 -> 3.4

**Pull request merged**